src/sqlfluff/dialects/dialect_databricks.py

```diff
@@ -12,6 +12,9 @@
 sparksql_dialect = load_raw_dialect("sparksql")
 databricks_dialect = sparksql_dialect.copy_as("databricks")
 
+databricks_dialect.sets("unreserved_keywords").update(
```

Comment:
this isn't needed as this set is already the spark unreserved keywords, due to the copy_as above